addons/mail/tools/discuss.py

```diff
@@ -10,3 +10,16 @@ def get_twilio_credentials(env):
     account_sid = params.get_param("mail.twilio_account_sid")
     auth_token = params.get_param("mail.twilio_account_token")
     return account_sid, auth_token
+
+
+def get_sfu_credentials(env) -> (str, str):
```

Comment:
I think we will need to split in 2 for easier override: `get_sfu_url` and `get_sfu_key`.